include/deal.II/dofs/dof_handler.h

```diff
@@ -370,20 +370,42 @@ class DoFHandler  :  public Subscriptor
                   const FiniteElement<dim,spacedim> &fe);
 
   /**
-   * Go through the triangulation and "distribute" the degrees of freedoms
-   * needed for the given finite element. "Distributing" degrees of freedom
-   * involves allocating memory to store the information that describes it
-   * (e.g., whether it is located on a vertex, edge, face, etc) and to
-   * sequentially enumerate all degrees of freedom. In other words, while the
-   * mesh and the finite element object by themselves simply define a finite
-   * element space $V_h$, the process of distributing degrees of freedom makes
-   * sure that there is a basis for this space and that the shape functions of
-   * this basis are enumerated in an indexable, predictable way.
+   * Go through the triangulation and "distribute" the degrees of
+   * freedoms needed for the given finite element. "Distributing"
+   * degrees of freedom involves allocating memory to store the
+   * indices on all entities on which degrees of freedom can be
+   * located (e.g., vertices, edges, faces, etc) and to then enumerate
```

Comment:
While you've touched this line, we could take the opportunity to make a pedantic grammatical correction:
etc -> etc.